### DISCUSSION TITLE
Windows 7 and Windows 2008 R2 are no longer supported.

### DISCUSSION THREAD

**Discussion by zhangbo8418** [rustdesk/rustdesk#4819] [Q&A]
Windows 7 and Windows 2008 R2 are no longer supported.

**Comment by zhangbo8418** [comment]
![M`G6M}PPMM15H4G)V_9YKRY](https://github.com/rustdesk/rustdesk/assets/46797568/b95870fb-7945-4cb7-973c-c68c1ee118f8)

**Comment by rustdesk** [comment]
no need to create duplicate

**Comment by rustdesk** [comment]
fixed, BTW do you have this problem on your win10? https://github.com/rustdesk/rustdesk/issues/4824

**Reply by zhangbo8418** [reply]
There are no issues with Win7, Win10, and Win11.

**Comment by zhangbo8418** [comment]
> fixed, BTW do you have this problem on your win10? #4824

no problem

**Comment by rustdesk** [comment]
Thanks

**Comment by zkteco-home** [comment]
![image](https://github.com/rustdesk/rustdesk/assets/66519168/71cc14f2-da69-495f-86c0-6fc118565c76)
![image](https://github.com/rustdesk/rustdesk/assets/66519168/c0f40fa7-3e98-4d27-a73c-3b529341d516)
just now i downloaded it again,same issue happend,but i have other windows 10,it is ok,my win10 version is:

![image](https://github.com/rustdesk/rustdesk/assets/66519168/c45b1750-9f61-46c1-b3f9-0d30762653f0)
last month i installed it from microsoft website

**Comment by zkteco-home** [comment]
I think you can close it,it is weird issue
